code/datums/traits/negative.dm

```diff
@@ -154,3 +154,12 @@
 	else if(prob(0.5) && dumb_thing)
 		to_chat(H, "<span class='danger'>You think of a dumb thing you said a long time ago and scream internally.</span>")
 		dumb_thing = FALSE //only once per life
+
+/datum/trait/dnr
+	name = "DNR"
+	desc = "Your genetic makeup is unstable, so much so that after death, you can't be revived through any known means"
```

Comment:
Recommended replacement:
`Your genetic makeup is unstable - so much so that all known means of revival are incapable of working on you. If you die, you're dead for good.`